src/components/FormFields/FormFields.jsx

```diff
@@ -1,21 +1,45 @@
-import './FormFields.scss'
+import "./FormFields.scss";
+import { useState } from "react";
+import error from '../../assets/Icons/error-24px.svg'
 
 const FormFields = (props) => {
+  const [change, setChange] = useState("");
+
+  const changeCounter = (event) => {
+    setChange(event.target.value);
+  };
+
+  console.log(props.valid);
+
   return (
     <div className="form">
       <div className="form__field-div">
-        <label
-          className="form__field-label"
-          htmlFor={props.inputName}
-        >
+        <label className="form__field-label" htmlFor={props.inputName}>
           {props.label}
         </label>
-        <input
-          className="form__field-input"
-          placeholder={props.label}
-          type="text"
-          name={props.inputName}
-        />
+        {change === "" && props.valid === false ? (
```

Comment:
a ternary for this looks so busy/confusing. Maybe a `if(change === "" && props.valid === false)` would look better?